src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -289,13 +320,17 @@ protected function createOutputCsv($outputTable)
         return new CsvFile($this->getOutputFilename($outputTable));
     }
 
-    protected function createManifest($table)
+    /**
+     * @param array $table
+     * @return bool|int
+     */
+    protected function createManifest(array $table)
```

Comment:
this could benefit from php-component methods as well... JFYI

Reply:
Also bool|int return is not ideal. Imho should have void|throw exception, but that's out of scope of the PR